src/Raven.Pal/build-all-windows.bat

```diff
@@ -13,18 +13,38 @@ cmd.exe /c ""%vcbin%\amd64\vcvars64.bat" & "%vcbin%\amd64\cl" -Felibrvnpal.x64.d
 copy librvnpal.x64.dll artifacts\librvnpal.win.x64.dll
 del *.obj librvnpal.x*
 
-echo =================================================
+cmd.exe /c ""%vcbin%\vcvars32.bat" & "%vcbin%\cl" -Felibrvnpal.win7x86.dll -I inc /D RVN_WIN7 /D WINVER=0x0601 /D _WIN32_WINNT=0x0601 /D NTDDI_VERSION=0x6010000 /D WINNT=1 /O2 /analyze /sdl /LD src\*.c src\win\*.c /link"
```

Comment:
I think it would be best to add those `WINVER` (and friends) to the other DLLs. Those need to match Win8.

Reply:
Another dll for Win8 or do you want to add WIN8 flags also for WIN10 ?

Reply:
single dll, exactly the one that we have now, but with correct flags, so we will not add features that we do not have in the future.